### PR TITLE
[web] Move embedding example out of experimental.

### PR DESCRIPTION
**Pull request by ditman**
Web embedding is now a reality, so let's move it out of `experimental`.

This PR only moves code from experimental to a new directory: `web_embedding`, and adds a small README file.

A new example app `ng-flutter` will land next.

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Contributors Guide]: https://github.com/flutter/samples/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Comment by ditman** [comment]
Waiting to see what CI has to say about this!

**Comment by domesticmouse** [comment]
Target the `beta` branch. This can't land until Flutter 3.10 is stable.

**Comment by ditman** [comment]
> Target the `beta` branch. This can't land until Flutter 3.10 is stable.

roger that!

**Comment by ditman** [comment]
This is landing way more than I want, let me rebase properly and force-push.

**Comment by domesticmouse** [comment]
Ping me when this is ready to review and merge into `beta` branch =)

File: tool/flutter_ci_script_beta.sh
Comment (ditman) [inline tool/flutter_ci_script_beta.sh:31]: Nope, try again!

File: tool/flutter_ci_script_master.sh
Comment (ditman) [inline tool/flutter_ci_script_master.sh:68]: @domesticmouse I think this is ready to land in `beta`. I'm not sure if this particular change is needed *now* or when beta lands into master. I guess CI will tell :)

**Comment by ditman** [comment]
CI Failures seem [similar to what's currently happening in the `beta` branch](https://github.com/flutter/samples/actions/runs/4808856119)?

File: tool/flutter_ci_script_master.sh
Comment (domesticmouse) [reply tool/flutter_ci_script_master.sh:68]: Can you also add this entry to stable CI? Yes it will fail, that's expected.

File: web_embedding/README.md
Comment (domesticmouse) [inline web_embedding/README.md:7]: Is this additional sample in a follow up PR?

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
> CI Failures seem [similar to what's currently happening in the `beta` branch](https://github.com/flutter/samples/actions/runs/4808856119)?

Sorting out CI issues is my thing of late. Ignore the stable CI failures, they are expected. I'll hopefully get beta and master CI sorted soon, then we can land this PR into `beta` branch with confidence

File: web_embedding/README.md
Comment (ditman) [reply web_embedding/README.md:7]: Yes, I'll do another PR to add that sample. I want to make sure that the commit that adds the sample is co-authored-by Mark.

**Comment by ditman** [comment]
>  I'll hopefully get beta and master CI sorted soon, then we can land this PR into beta branch with confidence

@domesticmouse pls clarify; should I land this PR as is, or wait until you get `beta` fixed and this one rebased with the fixes?

**Comment by domesticmouse** [comment]
> >  I'll hopefully get beta and master CI sorted soon, then we can land this PR into beta branch with confidence
> 
> 
> 
> @domesticmouse pls clarify; should I land this PR as is, or wait until you get `beta` fixed and this one rebased with the fixes?

Leave this PR as is. I'll land it soon. I may ask you to merge from upstream if I can't modify the PR directly 😋

**Comment by ditman** [comment]
The commit to land `ng-flutter` with correct attribution to @MarkTechson is here:

https://github.com/flutter/samples/compare/beta...ditman:publish_ng_flutter_sample?expand=1

That'll be the "second part" of this PR, after this lands.

**Comment by ditman** [comment]
(Rebased)

File: tool/flutter_ci_script_master.sh
Comment (ditman) [reply tool/flutter_ci_script_master.sh:70]: I almost miss this commit. I've fixed the `flutter_ci_script_stable.sh` entry as well!

**Comment by domesticmouse** [comment]
Please merge in the latest from `beta`. That should, in theory, clear most of the CI redness.

**Comment by domesticmouse** [comment]
> The commit to land `ng-flutter` with correct attribution to @MarkTechson is here:
> 
> 
> 
> https://github.com/flutter/samples/compare/beta...ditman:publish_ng_flutter_sample?expand=1
> 
> 
> 
> That'll be the "second part" of this PR, after this lands.

Please go ahead with creating this PR

**Comment by ditman** [comment]
Creating second PR right away. Thanks for landing this @domesticmouse!

**Pull request merged**
